web/modules/custom/asset_release/src/Controller/AssetReleaseController.php

```diff
@@ -73,18 +74,29 @@ public function add(RdfInterface $rdf_entity) {
    *   The access result object.
    */
   public function createAssetReleaseAccess(RdfInterface $rdf_entity) {
-    // Check that the passed in RDF entity is a collection, and that the user
-    // has the permission to create asset_releases.
-    // @todo Collection owners and facilitators should also have the right to
-    //   create asset_releases for the collections they manage.
-    // @see https://webgate.ec.europa.eu/CITnet/jira/browse/ISAICP-2448
-    if ($rdf_entity->bundle() == 'solution' && $this->currentUser()
-        ->hasPermission('create asset_release rdf entity')
-    ) {
-      return AccessResult::allowed();
+    $user = $this->currentUser();
+    if ($user->isAnonymous()) {
+      return AccessResult::neutral();
     }
```

Comment:
It should not be necessary to check if the user is anonymous. `Og::getMembership()` only returns memberships for members, so the `!empty($membership)` test on the next line already covers it.